### ISSUE TITLE
Add Web site for free hosting

### ISSUE THREAD

**Issue by arpit456jain** [arpit456jain/Web-Development-Path-And-Resources#7]

**Comment by harshved** [comment]
I will work on these issue

**Comment by arpit456jain** [comment]
ok @harshved

**Comment by harshved** [comment]
@arpit456jain Exactly, I have to write the steps to host the website in free hosting right?

**Comment by arpit456jain** [comment]
no @harshved you have to add a section like free hosting and add websites which provide free hosting like

for front end :  1 github pages 2. netlify
2. for python -: heroku and all
some thing like this 

and make sure you make  a table  with 2 columns site , Description

**Comment by anshkhandelwal12** [comment]
I Think the issue is not solved yet so ican solved this issue at complete this issue

**Comment by arpit456jain** [comment]
@anshkhandelwal12 go ahead
